reprojection.cpp

```diff
@@ -162,6 +164,42 @@ void reprojection::reproject(double *lat, double *lon) const
     *lon = x[0];
 }
 
+void reprojection::target_to_tile(double *lat, double *lon) const
```

Comment:
Please add a comment above explaining what the function does, see ``reprojection::coords_to_tile`` for an example